api/src/main/java/com/epam/pipeline/manager/cluster/performancemonitoring/ESMonitoringManager.java

```diff
@@ -66,6 +71,26 @@
                 : Collections.emptyList();
     }
 
+    @Override
+    public long getPodDiskSpaceAvailable(final String nodeName, final String podId, final String dockerImage) {
+        final MonitoringStats.DisksUsage.DiskStats diskStats =
+                AbstractMetricRequester.getStatsRequester(ELKUsageMetric.POD_FS, client)
+                        .requestStats(nodeName,
+                                DateUtils.nowUTC().minusMinutes(1L),
+                                DateUtils.nowUTC(),
+                                Duration.ofMinutes(1L)
```

Comment:
I suppose that we can reuse `cluster.monitoring.elastic.minimal.interval` preference which basically sets the minimal statistics interval. It is 1 minute by default.